docs/content/concepts/quickstart-tenancy-and-apis.md

```diff
@@ -42,16 +42,18 @@ workspaces                        ws           tenancy.kcp.io/v1alpha1
 
 ## Create and Navigate Some Workspaces
 
-The `ws` plugin for `kubectl` makes it easy to switch your `kubeconfig` between workspaces, and to create new ones:
+The `ws` plugin for `kubectl` makes it easy to switch your `kubeconfig` between workspaces and use `workspace create` plugin for `kubectl` to create new ones:
+> [!NOTE]  
+> Plugins can be found in the [release page](https://github.com/kcp-dev/kcp/releases/latest)
```

Comment:
It would be great if this note could link to https://docs.kcp.io/kcp/main/setup/kubectl-plugin/ (knowing that that page could also use some work) so we centralise where we talk about the kubectl plugins.